src/draft-ietf-rpp-requirements.md

```diff
@@ -85,11 +89,11 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT","SHOULD", "SH
 
 **R1.4.** RPP MUST include support for application level status codes, and MAY reuse the EPP status codes defined in [@!RFC5730].
 
-**R1.5.**
-RPP MUST include support for providing detailed information about application status codes, for example as described in [@!RFC7807]
+**R1.5.** RPP MUST include support for providing detailed information about application status codes, for example as described in [@!RFC7807]
+
+**R1.6** RPP MUST support additional information about a successful operation (information or warning) to convey additional information to the client for example about deprecation or partial success.
 
-**R1.6**
-RPP MUST support additional information about a successful operation (information or warning) to convey additional information to the client for example about deprecation or partial success.
+**R1.7** RPP MUST support both "thin" and "thick" registry models, and MUST allow for flexibility in the amount and type of data stored and returned by the server, according to the chosen registry model. This includes supporting mechanisms for indicating which data elements are available or omitted, and ensuring that clients can reliably determine the presence or absence of specific data fields based on the registry’s operational model.
```

Comment:
I think after it is defined as terms above you may skip quotation and use Thin registry and Thick registry as terms (capital letters).